product_docs/docs/pgd/5/admin-manual/installing/07-configure-proxies.mdx

```diff
@@ -19,7 +19,7 @@ It is best practice to configure PGD Proxy for clusters to enable this behavior.
 To set up a proxy, you will need to first prepare the cluster and sub-group the proxies will be working with by:
 
 * Logging in and setting the `enable_raft` and `enable_proxy_routing` node group options to `true` for the sub-group. Use [`bdr.alter_node_group_option`](/pgd/latest/reference/nodes-management-interfaces#bdralter_node_group_option), passing the sub-group name, option name and new value as parameters.
-* Create as many uniquely named proxies as you plan to deploy using [`bdr.create_proxy`](/pgd/latest/reference/routing#bdrcreate_proxy) and passing the new proxy name and the sub-group it should be attached to.
+* Create as many uniquely named proxies as you plan to deploy using [`bdr.create_proxy`](/pgd/latest/reference/routing#bdrcreate_proxy) and passing the new proxy name and the sub-group it should be attached to. The `bdr.create_proxy` sets default values for different proxy options like listen_port which defaults to 6432.
```

Comment:
The default value of 6432 will not be true after release of read scalability so should we add it?

Reply:
This is v5.4 guide. There will lots of changes I guess after read scalability. We can modify at that time or just mention something else like listen_address (0.0.0.0)

Reply:
What will the new default value be? And why is an existing default changing with the addition of read scalability?

Reply:
Changing it to`listen_address to `0.0.0.0``

Reply:
`listen_port` could be nullable after read scalability feature. listen_port will decide whether proxy is in default, read-only or any mode. We had added separate proxy mode to indicate that but after RFC discussion it was suggested to use this option for deciding proxy which mode proxy is configured in.